engine/src/main/java/org/terasology/logic/players/FirstPersonHeldItemMountPointComponent.java

```diff
@@ -40,7 +39,7 @@
     public float scale = 1f;
 
     // TODO: @In
-    private OpenVRProvider vrProvider = OpenVRProviderSingleton.vrProvider;
+    private final OpenVRProvider vrProvider = OpenVRProvider.getInstance();
```

Comment:
I must confess I am not well versed in the arcane arts of java timelines, i.e. what gets instantiated when. 

But I think by filling the vrProvider variable on declaration you might be forcing the OpenVRProvider instance to exist when the FirstPersonHeldItemMountPointComponent _class_ is loaded instead of waiting until an instance of the component is created. Tagging @flo for confirmation.

I'm guessing this might make a huge difference in this particular case but I'd create a default constructor and I'd initialize vrProvider in there.